src/index.css

```diff
@@ -0,0 +1,5 @@
+.h1 {
```

Comment:
The selector `.h1` is not semantically correct. Class names should not be named after HTML elements. Consider using a more descriptive class name.